src/main/kotlin/io/hemin/wien/util/CollectionsExtensions.kt

```diff
@@ -15,34 +15,34 @@ import io.hemin.wien.model.RssCategory
 
 /** Transforms this list into a list of [RssCategoryBuilder] */
 @JvmName("asRssCategoryBuilders")
-internal fun List<RssCategory>.asBuilders(): List<RssCategoryBuilder> = this.map(RssCategory.builder()::from)
```

Comment:
I have made these public as they're annotated with `@JvmName`

Reply:
The `@JvmName` annotation is just there to get rid of the type erasure issue. Wouldn't it the better to leave them internal and also annotate them with `@InternalApi`?

Reply:
I'm happy to, I have just misinterpreted what these were here for. Will do!